src/features/titles/plugin.js

```diff
@@ -0,0 +1,82 @@
+/* global wp */
+import React, { Fragment } from 'react';
+import { Panel } from './components';
+import { DEFAULT_TEST } from './data/shapes';
+import withTestData from './data/with-test-data';
+import Results from './results';
+import Settings from './settings';
+
+const {
+	PluginSidebar,
+	PluginSidebarMoreMenuItem,
+} = wp.editPost;
+const { __ } = wp.i18n;
+const { PanelBody } = wp.components;
+
+const Plugin = props => {
+	const {
+		post,
+		test,
+	} = props;
+	const {
+		started,
+		paused,
+		results,
+		end_time: endTime,
+	} = test;
+	const { winner } = results;
+
+	const classNames = [];
+
+	if ( started ) {
+		classNames.push( 'is-started' );
+	}
+
+	if ( paused ) {
+		classNames.push( 'is-paused' );
+	}
```

Comment:
Could condense these into `classNames = [ started && 'is-started', ... ]` then `classNames.filter( Boolean ).join( ' ' )`